src/adapter/src/catalog/builtin.rs

```diff
@@ -2344,17 +2344,34 @@ FROM
 pub const MZ_SHOW_CLUSTER_REPLICAS: BuiltinView = BuiltinView {
     name: "mz_show_cluster_replicas",
     schema: MZ_INTERNAL_SCHEMA,
-    sql: r#"CREATE VIEW mz_internal.mz_show_cluster_replicas
-AS SELECT
-    mz_catalog.mz_clusters.name AS cluster,
-    mz_catalog.mz_cluster_replicas.name AS replica,
-    mz_catalog.mz_cluster_replicas.size AS size
-FROM
-    mz_catalog.mz_cluster_replicas
-    JOIN mz_catalog.mz_clusters ON
-            mz_catalog.mz_cluster_replicas.cluster_id = mz_catalog.mz_clusters.id
-ORDER BY
-    1, 2"#,
+    sql: r#"CREATE VIEW
+    mz_internal.mz_show_cluster_replicas
+    AS
+        SELECT
+            mz_catalog.mz_clusters.name AS cluster,
+            mz_catalog.mz_cluster_replicas.name AS replica,
+            mz_catalog.mz_cluster_replicas.size AS size,
+            statuses.is_ready
+        FROM
+            mz_catalog.mz_cluster_replicas
+                JOIN
+                    mz_catalog.mz_clusters
+                    ON mz_catalog.mz_cluster_replicas.cluster_id = mz_catalog.mz_clusters.id
+                JOIN
+                    (
+                            SELECT
+                                replica_id,
+                                mz_internal.mz_all(
+                                        mz_internal.mz_cluster_replica_statuses.status
+                                        = 'ready'
+                                    )
+                                    AS is_ready
```

Comment:
As per our design principles, this should just be `ready` (https://www.notion.so/materialize/SQL-design-principles-2e2069797f7c46b8b4c7dd10043afdcd)